agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -622,7 +622,10 @@ public static class SamplingOverride {
 
     // TODO (trask) make this required when moving out of preview
     //   for now the default is both "request" and "dependency" for backwards compatibility
-    @Nullable public SamplingTelemetryKind telemetryKind;
+    @Nullable public SamplingTelemetryType telemetryType;
+
+    // this config option existed in one GA release (3.4.0), and was then replaced by telemetryType
+    @Deprecated @Nullable public SamplingTelemetryType telemetryKind;
```

Comment:
```suggestion
    @Deprecated @Nullable public SamplingTelemetryType telemetryType;
```

Reply:
this is the old (not deprecated) name, so that we don't break anyone who is already using it